ember-server/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -58,7 +59,7 @@ private[server] object ServerHelpers {
       idleTimeout: Duration = 60.seconds,
       additionalSocketOptions: List[SocketOptionMapping[_]] = List.empty,
       logger: Logger[F]
-  )(implicit C: Clock[F]): Stream[F, Nothing] = {
+  )(implicit C: Temporal[F]): Stream[F, Nothing] = {
```

Comment:
I usually avoid mixing context bonuds and named implicits.  Once I have to name one, I name them all.  I'd probably have an `F: Network[F]` (or maybe `N: Network[F]`) here.

I don't have a strong reason for that, and it was already that way when you found it, so I also don't mind being ignored here.